### ISSUE TITLE
Output log updates line-by-line instead of individual characters

### ISSUE THREAD

**Issue by xen-000** [ruarai/CompilePal#183]
**Describe the bug**

Previously, the output log would update whenever anything is printed, but now it only updates when a new line is printed. This means pacifiers ("1...2...3...") only show up when they're complete, making it hard to follow the progress in VVIS or VRAD.

**Comment by Exactol** [comment]
Thanks for reporting this, I'll fix it in the next update
